GNUmakefile.llvm

```diff
@@ -380,25 +386,25 @@ endif
 endif
 
 instrumentation/afl-llvm-common.o: instrumentation/afl-llvm-common.cc instrumentation/afl-llvm-common.h
-	$(CXX) $(CFLAGS) $(CPPFLAGS) `$(LLVM_CONFIG) --cxxflags` -fno-rtti -fPIC -std=$(LLVM_STDCXX) -c $< -o $@ 
+	$(CXX) $(CLANG_CPPFL) -c $< -o $@
```

Comment:
and why are you removing LLVM_STDCXX and -fPIC?
the plugins compile from llvm 3.7 onwards (except lto), and they must still work with your changes. 
same with your removal of -Wno-writable-strings ... your llvm version might not warn, but that flag is there for a reason.
llvm is quite volatile in its internal changes etc. little changes can break it.
so I can only merge if it either fixes a break - or improves and does not break something.